packages/portal/spec-renderer/src/components/SpecOperationsList.vue

```diff
@@ -154,7 +165,9 @@ const props = defineProps({
     type: String,
     default: '210',
   },
-
+  /**
+   * Used for component testing
+   */
```

Comment:
I don't see the `testMode` prop being utilized in the component anywhere, but it's passed as `true` in the component test. 

What is this prop doing? Sorry if I'm missing something